docs/user-guide/src/introduction.md

```diff
@@ -4,12 +4,50 @@
 
 # Metal³
 
-Metal³ project (pronounced: Metal Kubed) exists to provide components
-that allow you to do bare metal host management for Kubernetes. Metal³
-works as a Kubernetes application, meaning it runs on Kubernetes and is
-managed through Kubernetes interfaces.
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enrol your bare metal machines,
+provision operating system images, and deploy Kubernetes clusters to them. From
+there, operating and upgrading your Kubernetes clusters can be handled by
+Metal³. Moreover, Metal³ is itself a Kubernetes application, so it runs on
+Kubernetes, and uses Kubernetes resources and APIs as its interface.
 
 Metal³ is one of the providers for the Kubernetes sub-project [Cluster
-API](https://github.com/kubernetes-sigs/cluster-api) and is focused on
-providing Kubernetes native APIs to manage Kubernetes clusters in a bare
-metal environment.
\ No newline at end of file
+API](https://github.com/kubernetes-sigs/cluster-api). Cluster API provides
+infrastructure agnostic Kubernetes lifecycle management, and Metal³ brings the
+bare metal implementation. This is paired with one of the components from the
+OpenStack ecosystem, [Ironic](https://ironicbaremetal.org/), which supports a
+wide range of bare metal management protocols to boot and install machines, no
+matter the brand of hardware.
```

Comment:
This might be a good opportunity to mention that the reason we chose Ironic is that it has a mature community of hardware vendors, including CI testing on a variety of hardware.

Also it might pay to mention that Metal³ handles the deployment of Ironic, and that it operates in a standalone mode. (Any time OpenStack comes up it tends to frighten people.)

Reply:
Yeah, let's follow-up with this update.